spec/game_spec.rb

```diff
@@ -0,0 +1,102 @@
+require "spec_helper"
+require "tictactoe"
+
+RSpec.describe Game do
+  subject(:game) { Game.new }
+  let(:token) { "X" }
+
+  class Game::BoardWithOutcome
+    attr_reader :grid, :tokens
+
+    def initialize(grid)
+      @grid = grid
+      @tokens = {X: "X", O: "O"}
+    end
+
+    def get(position)
+      return tokens[:X] if grid.include?(position)
+      tokens[:O]
+    end
+  end
+
+  class Game::BoardWithNoOutcome
+    attr_reader :grid, :tokens
+
+    def initialize(grid)
+      @grid = grid
+      @tokens = {X: "X"}
+    end
+
+    def get(position)
+      return tokens[:X] if grid.include?(position)
+      nil
+    end
+  end
```

Comment:
These two test doubles are very similar, but I don't quite understand why they both need to exist. Is there a way to simplify or make the intention more revealing?